tests/test_simulation.py

```diff
@@ -3853,6 +3853,43 @@ def get_ref_per_body_results_skin(
 )
 
 
+def get_custom_scope(selection_name: str, mesh: MeshedRegion):
```

Comment:
Hi @janvonrickenbach I realize there are a lot of helper functions you made for tests. Do you think maybe some of them could be translated to helpers in the sources?